### ISSUE TITLE
REQUEST - miner runtime recording and reporting

### ISSUE THREAD

**Issue by madLyfe** [fireice-uk/xmr-stak#1135]
i brought this up with @fireice-uk awhile ago but i think it is needed for the miner to report operating time. im not sure how this would be reported in the interface but even something simple would be helpful. also, if someone wanted to use the running time data in their own tool, they could.
in xmr-stak you could report min/hr of runtime per week/month/year for example. this would give operators at least some hard data on actual mining time rather than the server just being on or times when the miner wasnt mining.

**Comment by DrStein99** [comment]
I agree, there should be a better way to track statistics, and more details.  As of now I have to write stuff down in books to take notes.   

I spent most of my career writing database processing software, in statistics.  I would like to make an interface to gather the stats, post them into mySQL tables.  This way there would be a database to analyze all kinds of data.

Unfortunately it is going to take me a lot of time to learn this console window style / universal C interface I'm not familiar yet with how to debug and run Visual Studio on a breakpoint level, yet.

**Comment by fireice-uk** [comment]
Self note:
- Implement by counting idling time
- CPU-time (4 cores at 1min = 4min) style might be actually desirable here.

**Comment by pulsarcmc** [comment]
Wouldn't be opposed to discussing this to get some bare minimum requirements. Could def. use PostgreSQL (libpq) and start adding some telemetry.
